Heisen jinja2_native (#75587)
ansible/ansible: docs/docsite/rst/porting_guides/porting_guide_core_2.13.rst

Playbook
========

> * Templating - it is no longer allowed to perform arithmetic and concatenation operations outside of the jinja template. The following statement will need to be rewritten to produce ``[1, 2]``:

Reviewer: ```suggestion
* Templating - You can no longer perform arithmetic and concatenation operations outside of the jinja template. The following statement will need to be rewritten to produce ``[1, 2]``:
```
